File: Test_Coverage/processed_dataset/tests/test_HumanEval_45.py
from functions.HumanEval_45 import triangle_area

def test_HumanEval_45():
     assert triangle_area(5, 3) == 7.5
     assert triangle_area(2, 2) == 2.0
     assert triangle_area(10, 8) == 40.0
 

File: Test_Coverage_Java/src/test/java/test_HumanEval_45.java
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class HumanEval_45Test {

    @Test
    public void test_HumanEval_45() {
        assertEquals(7.5, HumanEval_45.triangle_area(5, 3), 0.01);
        assertEquals(2.0, HumanEval_45.triangle_area(2, 2), 0.01);
        assertEquals(40.0, HumanEval_45.triangle_area(10, 8), 0.01);
    }
}